data/map.txt

```diff
@@ -27439,6 +27589,15 @@ planet "Graveyard Wormhole (Unstable)"
 planet "Graveyard Wormhole (Stable)"
 	description ``
 
+planet Greymoon
+	attributes station hai
+	landscape land/space6
+	description `This station is owned and operated by the Far Loo Bah fuel collection corporation. Their logo is emblazoned on nearly every possible surface and sign, indicating their sponsorship of every service and feature on the station. Interestingly, this includes a "History of Spaceflight" museum which takes up an entire branch of the station. Naturally, the museum focuses very heavily on the Far Loo Bah corporation's role in that history.`
+	spaceport `The spaceport consists of a wide, two-level promenade encircling the interior of the station, with the docking access opening directly onto the mezzanine. The upper level is mostly devoted to amenities, light commercial businesses, a medical center, and a bank. The lower level has all of the real businesses: multiple manufacturers, which are supplied by the heavier elements filtered out from the supply of fuel stock from the planet below, have store frontage here. A virtual outfitter provides sale options for locally-manufactured outfits that can be installed on your ship without you ever leaving the comfort of the promenade.`
```

Comment:
```suggestion
	spaceport `The spaceport consists of a wide, two-level promenade encircling the interior of the station, with the docking access opening directly onto the mezzanine. The bustling commerce hub features banks, businesses, wholesale vendors for various fuels, and even a virtual outfitter providing sale options for locally-manufactured outfits that can be installed on your ship without you ever leaving the comfort of the promenade.`
```

(too long, too much unnecessary detail)

Reply:
I profoundly disagree with this. It's a text based game, you don't exactly get a lot of detail from anywhere else...
The whole point of this was to create a mental image of a strict divide between two types of business, to set it apart from being "just another commerce hub". You've actively annihilated the intention there.

Reply:
Always ask yourself, "why does the player care?" In this case, why does the player care that the lower level of the promenade has the "real" businesses? How does this help the player? How does this fill in gaps in the player's knowledge about the Hai and their culture and history?

I don't see how the mention of a divide between types of businesses across different levels is useful or interesting information to the player. Can you explain how you see it differently?

Reply:
This connects to my other response: The player may not care when they read the description (though I enjoy finding the quirks in descriptions), but it provides a hook if someone wants to write content that occurs on the station, and gives them something specific to work with.

It also provides a little bit of differentiation between one place and another. I'm endeavouring to avoid having one thing feel too generically similar to another.

Reply:
If that's your goal, then I think you want to make the descriptions more unique, not less. Than means less of "this is how they've customized the default layout that gets repeated in multiple descriptions" and more of "this is the space station's uniqueness described for its own sake". Does that make sense?

Reply:
If I was adding stations built at all different parts of history then yes I would entirely agree with you, but the entire point is that these are the relics of a specific point in time when many Hai had to be relocated, so the underlying similarity is kind of the point. It's explicitly the intent to show how time has seen them customised from a similar starting point.